src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -2140,12 +2150,18 @@ export class SavedObjectsRepository {
    * @param type The type of the saved object.
    * @param id The ID of the saved object.
    * @param namespace The target namespace.
+   * @param initialNamespaces The target namespace(s) we intend to create the object in, if specified.
```

Comment:
nit: The difference between `namespace` and `initialNamespaces` is subtle and it might be worth adding a bit more info here about when each is used.